app-ng/src/globals.lisp

```diff
@@ -5,8 +5,8 @@
 
 (in-package #:qvm-app-ng)
 
-(defvar *logger* (make-instance 'cl-syslog:rfc5424-logger
-                                :app-name "qvm-ng"
-                                :facility ':local0
-                                :log-writer (cl-syslog:null-log-writer))
-  "The CL-SYSLOG logger instance.")
+(global-vars:define-global-var **available-simulation-methods** '(pure-state full-density-matrix)
```

Comment:
These seem constant, but defining constants is annoying for interactive dev. Maybe consider naming these `+x+` to signal they're constant, but you know it's a lie underneath it all.

Reply:
I almost did that here and elsewhere but chickened out! Now, with CodeReviewBlessings, I feel validated & safe.